### DISCUSSION TITLE
move purpur.yml to ./config folder

### DISCUSSION THREAD

**Discussion by hirata-estate** [PurpurMC/Purpur#1003] [__Dismissed__] [answered]
This seems to be a nice-to-have considering paper's changes to the paper.yml also being put in ./config.
The per-world & global file separations would be extremely nice too.

**Comment by BillyGalbreath** [comment, marked as answer]
We're in the process of doing just that. It's a big config, so it will take some time.

https://github.com/PurpurMC/Purpur/tree/dev/configurate-configs
